### PR TITLE
[UI] Add tab & dashboard summary Lit component

### PR DESCRIPTION
**Pull request by sultan-duisenbay**
Added a tab summary and dashboard summary Lit components.
Dashboard summary component is composed of tab summary components. Dashboard send the request to `/tab-summaries` API endpoint and populates individual tabs.

To run locally:
1. `sh bump-protos.sh` within *web* directory
2. start API via `bazel run  cmd/api:api -- -scope="gs://testgrid" --allowed-origin=*`
3. probably install lit and ts
4. `npm run build`
5. `npm run start`

Screenshot how the dashboard summary looks like
Screenshot from 2023-02-21 21-35-56

### CODE REVIEW

**Comment by sultan-duisenbay** [comment]
I'll add tests in the next PR :)

File: web/index.html [hunk 1]
```diff
@@ -21,7 +22,10 @@
 
 <body>
   <script type="module" src="./out-tsc/src/TestgridIndex.js"></script>
-  <testgrid-index></testgrid-index>
```
Comment (michelle192837) [inline web/index.html:24]: I don't think the <testgrid-index> element should be removed for index.html. Possibly this wants a separate .html file?

Comment (sultan-duisenbay) [reply web/index.html:24]: oh yeah, I was using index html to render the tab-summary, I left it as a comment and brought the original `testgrid-index` component back

Comment (michelle192837) [reply web/index.html:24]: Sounds good! Probably don't want to leave that long-term, but while we're developing this seems fine.

Comment (sultan-duisenbay) [reply web/index.html:24]: Yeah, we will have a single page app and all of those components will probably be inside of `testgrid-app` component in the future

File: web/custom-elements.json [hunk 2]
```diff
@@ -91,85 +91,843 @@
...
+        {
+          "kind": "function",
+          "name": "e"
+        },
+        {
+          "kind": "function",
+          "name": "e"
+        },
+        {
+          "kind": "function",
```
Comment (michelle192837) [inline web/custom-elements.json:413]: These additions seem a bit odd, was this auto-generated or is this expected?

**Review by michelle192837 — APPROVED** [review]
LGTM, adding a hold because of the weirdness in web/custom-elements.json. Unhold at will

/hold

File: web/custom-elements.json (hunk 2 above)
Comment (sultan-duisenbay) [reply web/custom-elements.json:413]: Yeah, this doesn't look right albeit I didn't do anything with it - it was auto-generated. I will checkout the newest version and see if it helps

Comment (sultan-duisenbay) [reply web/custom-elements.json:413]: so I checked out this file from main branch and then ran `npm build run`. This file additions were auto-generated.

**Pull request merged**